tests/unit/s2n_psk_offered_test.c

```diff
@@ -379,7 +379,7 @@ int main(int argc, char **argv)
 
         struct s2n_psk *psk2 = NULL;
         EXPECT_OK(s2n_array_pushback(&conn->psk_params.psk_list, (void**) &psk2));
-        EXPECT_OK(s2n_psk_init(psk2, S2N_PSK_TYPE_RESUMPTION));
```

Comment:
Did this fail before? I would have expected more modifications to the test if the init type was changed.

Reply:
Currently this test will fail without switching the psk type to resumption. The reason is that the resumption psk has no ticket_issue_time, so when we calculate the ticket age (current_time - ticket_issue_time = current_time) we get a value that is too large for the psk. So I could either set a ticket_issue_time for this psk or just change the type to external, given that the obfuscated_ticket_age is not actually being tested here.